pylint.cfg

```diff
@@ -1,4 +1,4 @@
 [MESSAGES CONTROL]
-disable=C0330,I1101,R0801,R0902,R0903,R0913,R0914,W1202,W1203,W1510
+disable=C0330,I1101,R0801,R0902,R0903,R0913,R0914,W1202,W1203,W1510,E1101,C2801
```

Comment:
I think globally disabling linter messages should be the exception (as it is, there were already too many). And `C2801` and particularly `E1101` are not messages that we would want to globally disable. Can you please remove these here and add `# pylint disable=E1101` comments to the lines where you _really_ need them? Better of course to avoid them entirely.